### ISSUE TITLE
Only activate if the player can attack you (combat lvls)

### ISSUE THREAD

**Issue by runeliteissues** [adhansen/wilderness-player-alarm#32]
as title says, add an option to only activate if the player has the ability to attack you, ie the correct combat lvl

**Comment by DapperMickie** [comment]
If I remember correctly, this has been suggested numerous times and discussed in the RuneLite Discord server, and this would not be accepted by RuneLite because of the 3rd party client post Jagex put out some time ago.

**Comment by adhansen** [comment]
@DapperMickie That is correct. This and filtering based on the skull status of other players are the two most commonly-requested rejected features.

**Comment by swirle13** [comment]
do you guys happen to have a link to the post you're referring to?

**Comment by adhansen** [comment]
There are guidelines both from Jagex and RuneLite maintainers that this plugin has to comply with:

From Jagex: https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1

From RuneLite: https://github.com/runelite/runelite/wiki/Rejected-or-Rolled-Back-Features

This feature falls under a RuneLite restriction:
> Level-based PvP player indicators (eg. highlighting players who can be attacked, or are within X levels range)

Another feature that I don't see explicitly mentioned on either page but that certainly wouldn't fly is alarming based on players' equipped gear. Such as ignoring naked players but alarming on mystics. Any new feature is going to be at RuneLite's discretion and this plugin likely receives a little extra scrutiny for already toeing the line of what's allowed.
